.gitignore

```diff
@@ -11,8 +11,5 @@ cf-ssh.yml
 # Ignore emacs autosave
 **/*~
 
-# Ignore configuration secrets file
-secret.rb
-
 # Ignore local envvars
-.env
```

Comment:
the new instructions are to use `.env.development` instead of `.env` locally. The reason for this is that I found that having env vars in a `.env` file as overriding env vars in`.env.test` and `.env.development` -- when I defined `CDX_USERNAME` in `.env`, for example, the env var spec failed. I suspect this is a load order issue but an easy way around it was to use env-specific env files.